website/src/get-started/wordcount-example.md

```diff
@@ -90,7 +90,7 @@ To view the full code in Python, see
 
 {:.language-go}
 To view the full code in Go, see
-**[wordcount_minimal.go](https://github.com/apache/beam/blob/master/sdks/go/examples/minimal_wordcount/minimal_wordcount.go).**
+**[minimal_wordcount.go](https://github.com/apache/beam/blob/master/sdks/go/examples/minimal_wordcount/minimal_wordcount.go).**
```

Comment:
FYI, I don't think the site uses this copy in beam just yet.

@melap Is that still true? Does this change need to go into the old site repo still?

Reply:
Correct, changes to website markdown still need to be done in beam-site until the migration is complete